tardis/energy_input/energy_source.py

```diff
@@ -350,3 +253,11 @@ def read_artis_lines(isotope, path_to_data):
         sep="  ",
         index_col=False,
     )
+
+
+def get_nuclear_lines_database(
+    path="~/Downloads/tardisnuclear/decay_radiation.h5",
```

Comment:
Is this general for everyone?  Does tardisnuclear always install here?

Reply:
Yes, but I have moved the path value to the main function as a kwarg so it is clearer for the user